packages/cells/src/widget.ts

```diff
@@ -1420,6 +1468,10 @@ export class MarkdownCell extends AttachmentsCell<IMarkdownCellModel> {
   constructor(options: MarkdownCell.IOptions) {
     super(options);
     this.addClass(MARKDOWN_CELL_CLASS);
+    this.node.setAttribute(
+      'aria-label',
+      this.transBundle.__(MARKDOWN_CELL_ARIA_LABEL)
```

Comment:
Quick comment: for the translation extractor to work it needs to match a specific regular expression and it includes `trans.__('string')` and the actual string needs to be inside of the the parentheses. I know it may sound disappointing that the translation tooling is not advanced enough to figure out the strings during runtime (well that would be difficult anyways) or using static analysis (this could work but apparently it was not written yet or did not become mainstream).

Long story short you will need to rename `transBundle` to `trans` and use the strings directly rather than referring to them by constants (or define the constants somewhere where you have access to `trans`).

Reply:
Thanks for the comment @krassowski, and for saving me a few hours of debugging! Is there any place this is documented (I checked READMEs in `@jupyterlab/translation*` and [API docs](https://jupyterlab.readthedocs.io/en/stable/api/modules/translation.html))? If not, can you please point me to where this can be added?

Reply:
I'm afraid the docs might be missing at this stage. @fcollonval and @goanpeca any thoughts?

Reply:
We definitely need to add that documentation. The best place seems the developer documentation of JupyterLab.

Reply:
@krassowski , I've resolved this comment - please check and add any other feedback you have. In my cursory search, I wasn't able to find any tests targeting aria content - let me know how we should proceed in this regard.

> Quick comment: for the translation extractor to work it needs to match a specific regular expression and it includes trans.__('string')

@krassowski / @fcollonval , can you please point me to this regex/related module ? I'll have a first-pass at adding documentation with this information.